src/diffusers/models/resnet.py

```diff
@@ -765,9 +764,7 @@ def forward(
                 self.conv_shortcut(input_tensor, scale) if not USE_PEFT_BACKEND else self.conv_shortcut(input_tensor)
             )
 
-        output_tensor = (input_tensor + hidden_states) / self.output_scale_factor
-
-        return output_tensor
+        return (input_tensor + hidden_states) / self.output_scale_factor
```

Comment:
Any reason for this change? :sweat_smile:

Reply:
Nope - rolled back :)